script.js

```diff
@@ -116,3 +139,70 @@ function sendMessage(groupId, message){
         error: function () { }
     });
 }
+
+
+function onConnectFailed(resp) {
+    console.log("onConnectFailed triggered");
+    if (navigator.onLine) {
+        console.log("reconnecting");
+        window.Applozic.ALSocket.reconnect();
+    }
+}
+function subscribeToTypingChannel(groupId){
+    window.Applozic.ALSocket.subscibeToTypingChannel(groupId);
+}
+
+function unsubscribeToTypingChannel(){
+    window.Applozic.ALSocket.unsubscibeToTypingChannel();
+}
+
+function subscribeToSupportChannel(){
+    window.Applozic.ALSocket.subscribeToSupportChannel();
+}
+function unsubscribeToSupportChannel(){
+    window.Applozic.ALSocket.unsubscribeToSupportChannel();
+}
```

Comment:
does not seems to be getting used

Reply:
subscribeToSupportChannel is used 
mentioned comment against other two